app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
While it is not completely bad, preferred way in Rails 4+ is to use `.find_by` and then passing in hash of arguments like `email: params[:email]` - so I would just suggest to stick with `.find_by`. The reason for this is `.find_by_email` is not very effective and goes through `method_missing` process and basically when you call `.find_by_email` it does not exist so it takes longer to perform.